nublado/templates/reaper/cronjob.template.yml

```diff
@@ -0,0 +1,72 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "nublado.fullname" . }}-reaper
+spec:
+  schedule: "{{.Values.reaper.minute_to_run}} {{.Values.reaper.hour_to_run}} * * *"
```

Comment:
can we just have this whole string be one variable?

Reply:
I trust people to get two integers right more than I trust them to get a five-token string containing stars right.